cypress/e2e/login/login.cy.ts

```diff
@@ -0,0 +1,84 @@
+const loginPageTestIds = {
+  inputEmail: 'login-form-email',
+  inputPassword: 'login-form-password',
+  submitButton: 'login-form-submit',
+  googleLoginButton: 'login-google',
+};
+
+const loginAriaLabels = {
+  loading: 'login-form-loader',
+};
+
+describe('Credential Login', () => {
```

Comment:
Prefer naming UI test with `verb + component` to describe a flow/feature from the user's perspective:

```suggestion
describe('Login with Credentials', () => {
```